From b10a49ab016ed9ac8f231845fb3e83fbf3505652 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 12 Oct 2020 14:36:35 +0100
Subject: [PATCH] ARM: bcm2711-rpi.dts: Unlock DMA channels 9 & 10

The downstream-specific override of the 32-bit DMA mask needlessly
disables channels 9 and 10 - the VPU doesn't use them. This override
is redundant, so delete it.

See: https://github.com/raspberrypi/linux/issues/3896

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi.dtsi | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

--- a/arch/arm/boot/dts/bcm2711-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2711-rpi.dtsi
@@ -246,13 +246,8 @@
 	};
 };
 
-&dma {
-	/* The VPU firmware uses DMA channel 11 for VCHIQ */
-	brcm,dma-channel-mask = <0x1f5>;
-};
-
 &dma40 {
-	/* The VPU firmware DMA channel 11 for VCHIQ */
+	/* The VPU firmware uses DMA channel 11 for VCHIQ */
 	brcm,dma-channel-mask = <0x7000>;
 };
 
